README.md

```diff
@@ -56,6 +56,16 @@ $ STANDALONE_HOST="$(ifconfig | grep "inet " | grep -v 127.0.0.1 | awk '{ print
 
 ```
 
+In Ubuntu, it should be
+```bash
+
+$ STANDALONE_HOST="$(ifconfig | grep "inet " | grep -v 127.0.0.1 | awk '{ print $2 }' | awk -F ':' '{ print $2 }' | awk 'NR==2')" j2 prometheus/standalone.yml.template > /tmp/standalone.prometheus.yml
+
+```
+
+If it still doesn't work properly, I suggest setting the IP manually.
```

Comment:
It is grammatically correct, we can refine it to be more concise and avoid using "suggest"/"recommend" in technical writing. The following refinement is for your reference. 
"If it does not work properly, you can set the IP manually."
If you adopt the suggestion, please check all similar cases.